Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -217,6 +219,23 @@ GameList::~GameList()
   Settings::GetQSettings().setValue(QStringLiteral("gridview/scale"), m_model.GetScale());
 }
 
+void GameList::SetWindowTheme(std::shared_ptr<const UICommon::GameFile> game)
+{
+  auto gameId = game.get()->GetGameID();
+  if (gameId == "ID-Project+ Launcher")
+  {
+    Settings::Instance().SetThemeName(QStringLiteral(PPLUS_THEME_DIR));
+  }
+  else if (gameId == "ID-vBrawl Launcher")
+  {
+    Settings::Instance().SetThemeName(QStringLiteral(VBRAWL_THEME_DIR));
+  }
+  else
+  {
+    Settings::Instance().SetThemeName(QStringLiteral(DEFAULT_THEME_DIR));
```

Comment:
@JaredWhiteOne if someone has a custom theme set, doing this will override their theme so maybe you need to store their previous theme (excluding the named themes that change based on the launcher) and set that one instead.